changelog/4.3.1_2024-07-22/4440

```diff
@@ -0,0 +1,6 @@
+Change: Bump target SDK to 34
+
+Target SDK was upgraded to 34 to keep the app updated with the latest android changes.
```

Comment:
`Android` should be in capitals. I'd not describe in that way, because we are not including changes within the new SDK, just upgrading the compile and target versions. This is our internal changelog, so, we don't have to hide the real cause. My suggestion:

`Target SDK was upgraded to 34 in order to fulfill Android platform requirements`

Reply:
Right, changed